client/dashboard/data/me.ts

```diff
@@ -1,5 +1,9 @@
 import wpcom from 'calypso/lib/wp';
 
+export enum UserFlags {
+	CALYPSO_ALLOW_NONPRIMARY_DOMAINS_WITHOUT_PLAN = 'calypso_allow_nonprimary_domains_without_plan',
+}
```

Comment:
We're talking about avoiding `enum`
p1754323932129889-slack-C0982082MSR

Can we achieve the same thing with

```ts
type UserFlags =
  | 'calypso_allow_nonprimary_domains_without_plan'
  | 'calypso_everything_else_____';
```